library/std/src/sys/windows/fs.rs

```diff
@@ -961,7 +961,9 @@ pub fn try_exists(path: &Path) -> io::Result<bool> {
             // `ERROR_SHARING_VIOLATION` means that the file has been locked by
             // another process. This is often temporary so we simply report it
             // as the file existing.
-            io::ErrorKind::Other if e.raw_os_error() == Some(c::ERROR_SHARING_VIOLATION as i32) => {
+            io::ErrorKind::Unknown
```

Comment:
```suggestion
            _
```

We should demonstrate best practices in the standard library.